core/embed/io/display/ltdc_dsi/display_gfxmmu.c

```diff
@@ -0,0 +1,61 @@
+#ifdef KERNEL_MODE
+#include <trezor_bsp.h>
+
+#include <sys/irq.h>
+
+#include "display_internal.h"
+
+extern uint8_t physical_frame_buffer_0[PHYSICAL_FRAME_BUFFER_SIZE];
+extern uint8_t physical_frame_buffer_1[PHYSICAL_FRAME_BUFFER_SIZE];
+
+extern const uint32_t gfxmmu_lut_config[2 * GFXMMU_LUT_SIZE];
+
+void display_gfxmmu_init(display_driver_t *drv) {
+  __HAL_RCC_GFXMMU_FORCE_RESET();
+  __HAL_RCC_GFXMMU_RELEASE_RESET();
+
+  /* GFXMMU clock enable */
+  __HAL_RCC_GFXMMU_CLK_ENABLE();
+
+  /* Enable GFXMMU interrupt */
+  NVIC_SetPriority(GFXMMU_IRQn, IRQ_PRI_NORMAL);
```

Comment:
Do we need to enable `GFXMMU_IRQ`, or are we missing `GFXMMU_IRQHandler()`?

Reply:
removed https://github.com/trezor/trezor-firmware/commit/9d02fa6dff7c8ab80535e64dfafef529d36b90d7